persistence-api/src/main/java/io/stargate/db/schema/SchemaBuilderImpl.java

```diff
@@ -345,14 +341,10 @@ public void indexCustom() {
   }
 
   @DSLAction
-  public void from(String fromVertex) {
-    this.fromVertex = fromVertex;
-  }
+  public void from(String fromVertex) {}
```

Comment:
This is probably out of scope of this PR but we should remove Graph methods completely. They are apparently no longer part of the DSL language (top-level class annotations).